napari/layers/points/points.py

```diff
@@ -273,7 +282,8 @@ def __init__(
         text=None,
         symbol='o',
         size=10,
-        edge_width=1,
+        edge_width=0.1,
```

Comment:
Do we need to change default here?

Reply:
This is the main reason why I wanted to add this: currently the default is fixed value of 1 data pixel. I imagine that most people like me only set the size, and more rarely use the edge width actively, which results in bad stuff happening every time your points have a size drastically different from 10-ish (almost invisible border, or border so big the point is invisible).

By exposing `edge_width_rel` and turning it on by default, we prevent these problems, while allowing people who actually *want* to set the edge width in data pixels to do so. Changing the default of `edge_width` to 0.1 is just necessary if the default of `edge_width_rel` is `True`, otherwise the edge width would be 100% of the size.

Reply:
ah nice, ok - i get it. makes sense to me, but maybe @jni can comment too